packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -265,6 +265,15 @@ const TRANSFORMS = {
       }
     }
 
+    const currentOperations = {}
+    const { $xapi } = obj
+    forEach(obj.current_operations, (operation, ref) => {
+      const task = $xapi.getObject(ref)
```

Comment:
This will throw if the object does not exist, if you don't want this behavior, you can pass a default value as second param.